owasp-top10-2017-apps/a8/amarelo-designs/deployments/docker-compose.yml

```diff
@@ -2,6 +2,8 @@ version: "2"
 services:
   app:
     container_name: app-a8 
+    environment:
+      JWT_SECRET: secret
```

Comment:
Oops! There's a credential in your code, @evandroferreiras! 
What do you think about using an `.env` file?